app/src/main/java/no/usn/mob3000/ui/components/settings/SettingsSectionApplication.kt

```diff
@@ -84,14 +86,14 @@ fun SettingsSectionApplication(
             onDismissRequest = { languageExpanded = false }
         ) {
             DropdownMenuItem(
-                text = { Text(stringResource(R.string.settings_section_app_language_enus)) },
+                text = { Text(stringResource(R.string.settings_section_app_language_enus),color = MaterialTheme.colorScheme.onBackground) },
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).